test/integration/node-specific/abort_signal.test.ts

```diff
@@ -751,9 +751,10 @@ describe('AbortSignal support', () => {
         ...args
       ) {
         if (args[1].find != null) {
+          await sleep(1);
```

Comment:
make it actually async

Reply:
Why is this necessary?  / do we need coverage where we test both the sync/async codepaths?